simplq/src/components/pages/Home/StaticInfos.jsx

```diff
@@ -49,3 +50,84 @@ export const ExtraInfo = () => (
     </div>
   </div>
 );
+
+export const Footer = () => (
+  <div className={styles.footer}>
```

Comment:
Let's standardise everywhere to use `[]`. 

`styles['footer']` instead of `styles.footer`

@maaverik what do you say?

Reply:
Sounds good. I think prettier autoformats it this way. We just need to disable that setting.

Reply:
@MehmetFiratKomurcu can you try? We can take care of prettier if it comes in the way

Reply:
done